I've just tried building this and get the following:
`[ERROR] Failed to execute goal on project flume-hdfs-sink: Could not resolve dependencies for project org.apache.flume.flume-ng-sinks:flume-hdfs-sink:jar:1.10.0-SNAPSHOT: The following artifacts could not be resolved: com.fasterxml.jackson.core:jackson-annotations:jar:2.9.10.3, com.fasterxml.jackson.core:jackson-core:jar:2.9.10.3: Could not find artifact com.fasterxml.jackson.core:jackson-annotations:jar:2.9.10.3 in central (https://repo.maven.apache.org/maven2)`

--- a/pom.xml
+++ b/pom.xml
@@ -62,7 +62,7 @@ limitations under the License.
     <curator.version>2.6.0</curator.version>
     <derby.version>10.14.1.0</derby.version>
     <elasticsearch.version>0.90.1</elasticsearch.version>
-    <fasterxml.jackson.version>2.9.10.3</fasterxml.jackson.version>
+    <fasterxml.jackson.version>2.9.10</fasterxml.jackson.version>
     <fest-reflect.version>1.4</fest-reflect.version>
     <geronimo-jms.version>1.1.1</geronimo-jms.version>
     <gson.version>2.2.2</gson.version>

